webapp/src/pages/Landing/Landing.tsx

```diff
@@ -38,7 +48,9 @@ export default function Landing() {
       <div className="landing-login-button">
         <Button
           variant={ButtonVariant.SUBMIT}
-          onClick={() => (window.location.href = LOGIN_EP_URL)}
+          onClick={() =>
+            window.location.replace(`${LOGIN_EP_URL}?state=${from}`)
```

Comment:
why is `location.replace(...)` a better idiom than `location.href`?

Reply:
`location.replace()` redirects to the URL and replaces the current resource with the one at the provided URL, meaning the user won't be able to use the back button to navigate to it. (Source [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Location/replace))

I think this is better because after the user login, they can't go back to the login page if they press the back button.